tests/test_codestyle.py

```diff
@@ -11,10 +11,11 @@ def test_conformance(self):
         # W292 no newline at end of file
         # E722 bare except
         style = pycodestyle.StyleGuide(quiet=False,
-                                       ignore=['E501', 'E731', 'W291', 'W391', 'W292', 'E722', 'E402'])
+                                       ignore=['E501', 'E731', 'W291', 'W504',
+                                           'W391', 'W292', 'E722', 'E402'])
         # style.input_dir('../../pya')
-        style.input_dir('./')
+        style.input_dir('./pya')
         # style.input_dir('tests')
```

Comment:
this will only check module code for style issues. Imho, tests should also comply to style guides.

Reply:
Using ./ risk also checking venv site-packages. if venv is created inside the package.

Reply:
should add './test' then